src/Symfony/Bundle/WebServerBundle/composer.json

```diff
@@ -26,6 +27,9 @@
             "/Tests/"
         ]
     },
+    "conflict": {
+        "symfony/dependency-injection": "<3.3"
```

Comment:
I think the goal for this bundle is to be able to reuse all the CLI code for Silex too (the DI part here is really just about registering the command services to make them available in Symfony), without creating both a component and a bundle.